datafusion/physical-plan/src/execution_plan.rs

```diff
@@ -467,6 +468,383 @@ pub trait ExecutionPlan: Debug + DisplayAs + Send + Sync {
     ) -> Result<Option<Arc<dyn ExecutionPlan>>> {
         Ok(None)
     }
+
+    /// A physical optimizer rule that pushes down filters in the execution plan.
```

Comment:
Here are some documentation suggestions:
- Suggestions for docs: https://github.com/pydantic/datafusion/pull/23